File: 01_IFSP/LP1/Reposicao_2aul/EX1-Empresa-r/EX1.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String nome;
        float sa, ns;
        int pct;

        System.out.print("Funcionário: ");
        nome = scanner.nextLine();

        System.out.print("Salario: ");
        sa = scanner.nextFloat();

        if (sa <= 400.00f) {
            pct = 15;
        } else if (sa <= 700.00f) {
            pct = 12;
        } else if (sa <= 1000.00f) {
            pct = 10;
        } else if (sa <= 1800.00f) {
            pct = 7;
        } else if (sa <= 2500.00f) {
            pct = 4;
        } else {
            pct = 0;
        }

        ns = sa + (sa * pct / 100);

        System.out.println("Nome do funcionário: " + nome);
        System.out.println("% de aumento: " + pct + "%");
        System.out.printf("Salário atual: R$ %.2f%n", sa);
        System.out.printf("Novo salário: R$ %.2f%n", ns);
    }
}

File: 01_IFSP/LP1/Reposicao_2aul/EX1-Empresa-r/EX1.py

nome = input("Funcionário: ")
sa = float(input("Salario: "))
if sa <= 400.00:
    pct = 15
elif sa <= 700.00:
    pct = 12
elif sa <= 1000.00:
    pct = 10
elif sa <= 1800.00:
    pct = 7
elif sa <= 2500.00:
    pct = 4
else:
    pct = 0
ns = sa + (sa * pct / 100)
print(f"Nome do funcionário: {nome}")
print(f"% de aumento: {pct}%")
print(f"Salário atual: R$ {sa:.2f}")
print(f"Novo salário: R$ {ns:.2f}")
